clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreBufferService.java

```diff
@@ -591,5 +592,10 @@ public long getPubSubMessageTime() {
     public int getPayloadSize() {
       return 0;
     }
+
+    @Override
+    public PubsubMessageHeaders getHeaders() {
+      return new PubsubMessageHeaders();
```

Comment:
Why does it need this? I don't get what it is overriding..

Reply:
`PubsubMessageHeaders` was added to `PubSubMessage` interface and here
`FakePubSubMessage implements PubSubMessage.

Removing headers from `PubsubMessageHeaders` for now as it is not being used.

Reply:
Hmm... I am a bit concerned by this. We use headers very infrequently, so I hope we can make them zero overhead in the vast majority of cases where we don't use them. We shouldn't carry headers around in every message.